tools/release_provenance/models.py

```diff
@@ -0,0 +1,50 @@
+from sqlalchemy import (
+    DateTime, Text, String, Column, ForeignKey )
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, sessionmaker
+
+USERNAME = ""
+PASSWORD = ""
+HOST = "ceqv5w67lzc.db.cloud.edu.au"
+DATABASE = "TEST_RELEASE_PROVENANCE"
+CONNSTR = f'postgresql://{USERNAME}:{PASSWORD}@{HOST}/{DATABASE}'
```

Comment:
Can these (or just the `CONNSTR`) be passed into the script via `stdin` instead of being constants?

Reply:
Just so we can have something like `save_release.py my.json ${{ secrets.BUILD_DB_CONNECTION_STR }}`

Reply:
I don't think it's a good idea to pass cred with `stdin`. As they are constants we can directly fetch the value from secrets in the script itself. One such example would be to use `os.environ['const_name']`. I haven't done that bc I wasn't sure which method you would prefer to access these constants or the naming convention you wanna follow in github secrets. I feel it is better if I leave it for you to edit later.

Reply:
Updated it so the secret is set on the runner and obtained via `BUILD_DB_CONNECTION_STR`